### DISCUSSION TITLE
Device parameters

### DISCUSSION THREAD

**Discussion by SenfMeister78** [bisand/node-red-contrib-panasonic-comfort-cloud#23] [Q&A]
Hi.
I finally got to a place in my Node-Red project, where it was time to play with the Panasonic heat pump.
I can see that you have described a large amount of parameters that I can set, and that would take me far. 
But when pulling data from my device, I got parameters that I don't know what they do.
Among others are 
clothesDrying: false
insideCleaning: false
fireplace: false
I have no clue what they do..  True or False.

**Comment by bisand** [comment]
Hi, I am not quite sure what those parameters mean. The parameters are just relayed from the Panasonic API. The parameters seem to differ from product to product. You might find some answer from the Comfort Cloud app. Maybe you see any values there that can give you any clues.
